changelogs/bundler_256.md

```diff
@@ -0,0 +1,3 @@
+## Ruby apps with Bundler 2.x now receive version 2.5.6
+
+The [Ruby Buildpack](https://devcenter.heroku.com/articles/ruby-support#libraries) now includes Bundler 2.5.6. Applications specifying Bundler 2.x in their `Gemfile.lock` will now receive this version of bundler.
```

Comment:
Leaving a note for anyone interested in when this is going to roll out. It is EOD for me today. I want to be around when this ships. I'll likely aim for around noon CST tomorrow. (Feb 22).